doc/desiInstall.rst

```diff
@@ -224,6 +222,9 @@ src
     :command:`desiInstall` will attempt to execute :command:`make -C src all`.  This build type
     *is* mutually exclusive with 'make', but is not mutually exclusive with
     the other types.
+plain
+    If no other build type is detected, the downloaded code
+    is simply copied to the final install directory.
 
 **It is the responsibility of the code developer to ensure that these
```

Comment:
Since the build types are now mutually exclusive, perhaps we should rephrase: "It is the responsibility of the code developer to understand these build types and choose the one appropriate for the package being developed."

Reply:
Good; rephrased with your suggestion.